### ISSUE TITLE
false alarm on string interpolation

### ISSUE THREAD

**Issue by JUSTIVE** [edvardchen/eslint-plugin-i18next#57]
```javascript
"table": Object {
      "addOnes": Object {
-       "totalCount": "ValidMessage<String>",
+       "totalCount": "String('total {{0}}cases') ===> Expected argNameOrNumber but '{' found.",
      },
    },
```

which is valid in [i18n-react's string interpolation syntax](https://www.i18next.com/translation-function/interpolation), marked as `argNameOrNumber` error here
